source/ssb-chart/templates/access-ingress-internal-authorization-policy.yaml

```
@@ -0,0 +1,97 @@
+{{- if and .Values.access (and .Values.access.ingress .Values.access.ingress.internal) }}
+{{- range .Values.access.ingress.internal }}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: "{{ template "app.name" $ }}-ingress-{{.namespace }}-{{ .application }}-auth-policy"
```

You can use "required" here or at the beginning of the "range" to ensure these are defined. 

```suggestion
  name: "{{ template "app.name" $ }}-ingress-{{ required "Missing namespace" .namespace }}-{{ required "Missing application" .application }}-auth-policy"
```